scss/_variables.scss

```diff
@@ -772,7 +772,7 @@ $dt-font-weight:              $font-weight-bold !default;
 
 $list-inline-padding:         $spacer * .25 !default;
 
-$mark-padding:                .1875em !default;
+$mark-padding:                0 .2em !default; // Boosted mod
```

Comment:
```suggestion
$mark-padding:                0 .1875em !default; // Boosted mod
```

Reply:
Sure, we can. It can indeed be more relevant that way than staying closer to the value we put in Boosted v4.6...
@julien-deramond @louismaximepiton: Tell me what you like better.